public/docs/_examples/toh-1/ts/app/app.component.ts

```diff
@@ -13,10 +13,14 @@ export class Hero {
   template:`
     <h1>{{title}}</h1>
     <h2>{{hero.name}} details!</h2>
-    <div><label>id: </label>{{hero.id}}</div>
+    <dl>
+      <dt>id:</dt>
+      <dd>{{hero.id}}</dd>
+    </dl>
     <div>
-      <label>name: </label>
-      <input [(ngModel)]="hero.name" placeholder="name">
+      <label>name: 
```

Comment:
There I fully agree with you. This exercise is exactly about giving **a11y** that position but to find the balance where the lessons of the separate docs are not diluted.